tests/unittests/QuantizationTest.cpp

```diff
@@ -974,15 +990,16 @@ static Function *createGRUForQuantization(Module *M,
 TEST_P(Operator, end2endGRU) {
   // STEP1 - Generate the first network to record the quantization parameters.
   auto *mod = &profileEE.getModule();
+  auto *modBackend = &backendSpecificEE.getModule();
   PlaceholderBindings bindings;
-  Function *F1 = createGRUForQuantization(mod, bindings, "main");
-  Function *F2 = F1->clone("main2");
```

Comment:
Is `clone` totally useless now, since there's no way to "inject" a Function into an EE?

Reply:
It's still used elsewhere, e.g. in `cloneFunInsideFun()` where a function is replicated inside itself many times for parallel testing purposes.

Additionally we talked previously about adding an ability to clone a Module, I don't recall the exact reason we decided against it for now, but would be useful there too.